server/app_env/production.py

```diff
@@ -16,4 +16,5 @@
 
 
 class Config(_base.Config):
-  GA_ACCOUNT = 'UA-117119267-1'
\ No newline at end of file
+  GA_ACCOUNT = 'UA-117119267-1'
+  SHOW_DISASTER = True
```

Comment:
Do we want to make it "True" by default?

Reply:
done